metecho/api/gh.py

```diff
@@ -55,6 +55,20 @@ def gh_as_app(repo_owner, repo_name):
     return gh
 
 
+def get_org(orgname: str):
+    """
+    Authenticate as a GitHub organization.
+    A different app key is used to grant write access.
```

Comment:
Can we name this something like get_org_for_repo_creation to minimize the chance that we accidentally use it for something else later?

Reply:
Fixed in 9cd8092e7bae2c15f3a86ba368a491529ebcb506